src/devtools/client/webconsole/actions/input.ts

```diff
@@ -90,18 +86,21 @@ export function paywallExpression(expression: string, reason = "team-user"): UIT
 let nextEvalId = 1;
 export function evaluateExpression(expression: string): UIThunkAction {
   return async (dispatch, getState, { ThreadFront }) => {
-    const debuggerPanel = window.gToolbox.getPanel("debugger");
-
     if (!expression) {
       expression = window.jsterm?.editor.getSelection();
     }
-    if (!expression || !debuggerPanel) {
+    if (!expression) {
       return null;
     }
 
-    const { asyncIndex, frameId } = debuggerPanel.getFrameId();
```

Comment:
we can replace `debuggerPanel.getFrameId()` with `getSelectedFrame(getState());`